Libraries/LibJS/Tests/while-basic.js

```diff
@@ -0,0 +1,21 @@
+load("test-common.js");
+
+try {
+    var number = 0;
+    while (number < 9) {
+        number++;
+    }
+    assert(number === 9);
+
+    number = 0;
+    while(number < 3) number++;
+    assert(number === 3);
+
+    while (false) {
+        assertNotreached();
```

Comment:
who's responsible for this sin against humanity?
`Notreached`

Reply:
At least that shows the `while` statement working as expected :smile: